python/composio/tools/toolset.py

```diff
@@ -67,22 +67,28 @@
 P = te.ParamSpec("P")
 
 _KeyType = t.Union[AppType, ActionType]
-_ProcessorType = t.Callable[[t.Dict], t.Dict]
+_CallableType = t.Callable[[t.Dict], t.Dict]
```

Comment:
Consider using a more specific type for `_CallableType`. Instead of `t.Callable[[t.Dict], t.Dict]`, you could use `t.Callable[[t.Dict[str, t.Any]], t.Dict[str, t.Any]]` to be more precise about the input and output types.